admin/acceptance.py

```diff
@@ -253,10 +262,26 @@ def stop_cluster(self, reactor):
         return succeed(None)
 
 
-def generate_certificates(nodes):
+def _provider_for_cluster_id(dataset_backend):
+    """
+    Get the ``Providers`` value that probably corresponds to a value from
+    ``DatasetBackend``.
+    """
+    if dataset_backend is DatasetBackend.aws:
+        return Providers.AWS
+    if dataset_backend is DatasetBackend.openstack:
+        return Providers.OPENSTACK
+    return Providers.UNSPECIFIED
```

Comment:
Looks like a good enough working solution for now, until we unify Provider definition across code base. Thanks for the quick fix!